tests/integration/importExport/utils/compare.ts

```diff
@@ -54,6 +54,21 @@ export const expectSameSyncEnvironments = async (
   expectSameAllEnvData(data1, data2, { include: ["types", "snippets", "taxonomies"] });
 };
 
+export const expectSameSyncContentEnvironments = async (
```

Comment:
The function looks pretty specific to the sync content command. I'd consider moving it to the sync content integration tests file and making it local to them. If you agree, could you also move the `expectSameSyncEnvironments` above to the sync model tests?